.projenrc.ts

```diff
@@ -28,6 +28,9 @@ const project = new CdklabsConstructLibrary({
     packageManager: NodePackageManager.YARN_CLASSIC,
     prettier: true,
     prettierOptions: p.prettierOptions,
+    githubOptions: {
+        mergeQueue: true
+    },
```

Comment:
use this instead:
```suggestion
    enablePRAutoMerge: true,
```

Reply:
Swapped over to this - looks like it synthed the same (using Auto Queue). Thank you!